### ISSUE TITLE
Vertical scroll/drag on mobile with a hammer element on the page

### ISSUE THREAD

**Issue by calabresechris** [hammerjs/hammer.js#1029]
There are a million similar questions but I can't find an answer that works for me.

Here is the situation:

I have an HTML page, and within that page is an element that I am using hammer.js on.

Need to be able to scroll like this:
![image](https://cloud.githubusercontent.com/assets/15618943/19010230/bc62a9be-8731-11e6-9cd4-15473589c09c.png)
--->
![image](https://cloud.githubusercontent.com/assets/15618943/19010234/cfdcfa26-8731-11e6-9556-917e97611f32.png)

While also being able to pinch-to-zoom (and subsequently pan on that zoomed element) on the seating chart element above.

The element itself works perfectly. I'm using `doubletap`, `pinch`, `pinchend`, `pan`, and `panend` on it.

Now, in the event that the element is totally zoomed out (I'm keeping track of the scale for this reason), I would like the entire page to scroll when using it on a mobile browser (aka the finger will be dragging the page up). 

I have tried almost everything under the sun at this point. I can't seem to get it to manually scroll to a specific position (I have tried setting `window.scrollTop` and using `window.scrollTo()` with no results). 

If someone could point me in the right direction, I'll worship you and your family for the next...say....13 days. Heck, maybe even 14. 

**TL;DR**

- Have we pinch zoomed on the element? If so, handle panning around that element with glee!
- Are we fully zoomed out / pinched out on the element? If so, mobile users should be able to scroll the page like normal!

Thanks
Chris

**Comment by runspired** [comment]
This is a question best asked on SO or in the hammer slack.
